config/etc/system-image/writable-paths

```diff
@@ -13,6 +13,7 @@
 /tmp                                    none                    temporary   none        defaults
 /mnt                                    none                    temporary   none        defaults
 /media                                  none                    temporary   none        defaults
+/boot/androidboot                       auto                    persistent  none        none
```

Comment:
I do not feel very comfortable with the idea of modifying the kernel cmdline of the running boot partition. The advantages of using instead an env file are:

1. No need to add abootimg to the core snap
2. Independence from boot.img format. If the vendor did modifications to the standard format we can handle that in recovery, which can contain device-specific quirks.
3. It is more similar to what we do for grub and u-boot.
4. Touching current boot.img seems more risky as when you are upgrading you want to be as safe as possible and not modify the currently running kernel if possible

Besides that, I agree with the approach as you draft it.